[PyTorch] Remove implicit conversion from Tuple to vector reference

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64066
* #64065
* #64277
* #64110
* __->__ #63993
* #64031

This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

swolchok committed:
[PyTorch] Remove implicit conversion from Tuple to vector reference

This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

lw reviewed (approved):
I'm certainly not the right person to review this, you should perhaps ask someone on the JIT team. However since this just removes code and builds are passing I can stamp it. (Note though that one build job on CircleCI failed).

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Remove implicit conversion from Tuple to vector reference"


This seems to be unused, and it's pretty scary.

Differential Revision: [D30560441](https://our.internmc.facebook.com/intern/diff/D30560441/)

[ghstack-poisoned]
